worlds/rogue_legacy/Items.py

```diff
@@ -1,111 +1,389 @@
+    "Sky Blueprints":           RLItemData(ItemClassification.useful,        47, get_blueprints_quantity),
+    "Dragon Blueprints":        RLItemData(ItemClassification.useful,        48, get_blueprints_quantity),
+    "Slayer Blueprints":        RLItemData(ItemClassification.useful,        49, get_blueprints_quantity),
+    "Blood Blueprints":         RLItemData(ItemClassification.useful,        50, get_blueprints_quantity),
+    "Sage Blueprints":          RLItemData(ItemClassification.useful,        51, get_blueprints_quantity),
+    "Retribution Blueprints":   RLItemData(ItemClassification.useful,        52, get_blueprints_quantity),
+    "Holy Blueprints":          RLItemData(ItemClassification.useful,        53, get_blueprints_quantity),
+    "Dark Blueprints":          RLItemData(ItemClassification.useful,        54, get_blueprints_quantity),
 
     # Runes
-    "Vault Runes":              RLItemData("Runes",      90_060, ItemClassification.progression),
-    "Sprint Runes":             RLItemData("Runes",      90_061, ItemClassification.progression),
-    "Vampire Runes":            RLItemData("Runes",      90_062, ItemClassification.useful),
-    "Sky Runes":                RLItemData("Runes",      90_063, ItemClassification.progression),
-    "Siphon Runes":             RLItemData("Runes",      90_064, ItemClassification.useful),
-    "Retaliation Runes":        RLItemData("Runes",      90_065),
-    "Bounty Runes":             RLItemData("Runes",      90_066),
-    "Haste Runes":              RLItemData("Runes",      90_067),
-    "Curse Runes":              RLItemData("Runes",      90_068),
-    "Grace Runes":              RLItemData("Runes",      90_069),
-    "Balance Runes":            RLItemData("Runes",      90_070, ItemClassification.useful),
-
-    # Junk
-    "Triple Stat Increase":     RLItemData("Filler",     90_030, weight=6),
-    "1000 Gold":                RLItemData("Filler",     90_031, weight=3),
-    "3000 Gold":                RLItemData("Filler",     90_032, weight=2),
-    "5000 Gold":                RLItemData("Filler",     90_033, weight=1),
+    "Vault Runes":              RLItemData(ItemClassification.progression,   60),
+    "Sprint Runes":             RLItemData(ItemClassification.progression,   61),
+    "Vampire Runes":            RLItemData(ItemClassification.useful,        62),
+    "Sky Runes":                RLItemData(ItemClassification.progression,   63),
+    "Siphon Runes":             RLItemData(ItemClassification.useful,        64),
+    "Retaliation Runes":        RLItemData(ItemClassification.filler,        65),
+    "Bounty Runes":             RLItemData(ItemClassification.filler,        66),
+    "Haste Runes":              RLItemData(ItemClassification.filler,        67),
+    "Curse Runes":              RLItemData(ItemClassification.filler,        68),
+    "Grace Runes":              RLItemData(ItemClassification.filler,        69),
```

Comment:
I don't understand this being `filler`, Grace Runes have seemed like probably the most powerful item in the game. I'll go from dying constantly to a basically free win after equipping 5 of these.